output/opensearch.md

```diff
@@ -0,0 +1,163 @@
+# opensearch
+
+The `out_opensearch` Output plugin writes records into Opensearch. By default, it creates records using [bulk api](https://opensearch.org/docs/latest/opensearch/rest-api/document-apis/bulk/) which performs multiple indexing operations in a single API call. This reduces overhead and can greatly increase indexing speed. This means that when you first import records using the plugin, records are not immediately pushed to OpenSearch.
+
+Records will be sent to OpenSearch when the `chunk_keys` condition has been met. To change the output frequency, please specify the `time` in `chunk_keys` and specify `timekey` value in the configuration.
+
+This document does not describe all the parameters. For details, refer to the **Further Reading** section.
+
+## Installation
+
+Since `out_opensearch` has been included in the standard distribution of `td-agent` since v4.4.0 and the alternative distribution of `calyptia-fluentd` since v1.3.4, `td-agent`/`calyptia-fluentd` users do not need to install it manually.
+
+If you have installed Fluentd without `td-agent` or `calyptia-fluentd`, please install this plugin using `fluent-gem`:
+
+```text
+$ fluent-gem install fluent-plugin-opensearch
+```
+
+## Example Configuration
+
+Here is a simple working configuration which should serve as a good starting point for most users:
+
```

Comment:
It may be obvious, but it is kind to note "for OpenSearch instance on localhost" or something.

Reply:
Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/390/commits/798582a31dffd7c5dc2a71ccc267e4ac37ced8e2.